package.json

```diff
@@ -53,5 +53,13 @@
     "traverse": "^0.6.6",
     "uuid": "^3.0.1"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "ci": {
+    "downstreamIgnoreList": [
+      "loopback-connector-db2z",
+      "loopback-connector-informix",
+      "loopback-connector-mqlight",
+      "loopback-connector-cassandra"
```

Comment:
I assume this is the only way to tell CI to run `loopback-connector-cassandra` with juggler3.x only since [juggler3.x](https://github.com/strongloop/loopback-datasource-juggler/blob/master/package.json#L56) does not ignore cassandra.